.travis.yml

```diff
@@ -19,3 +22,12 @@ notifications:
   email: false
   slack:
     secure: VkbZLIc2RH8yf3PtIAxUNPdAu3rQQ7yQx0GcK124JhbEnZGaHyK615V0rbG7HcVmYKGPdB0cXqZiLBDKGqGKb2zR1NepOe1nF03jxGSpPq8jIFeEXSJGEYGL34ScDzZZGuG6qwbjFcXiW5lqn6t8igzp7v2+URYBaZo5ktCS2xY=
+addons:
+  coverity_scan:
+    project:
+      name: "intelsdi-x/snap-plugin-collector-rdt"
+      description: "Build submitted via Travis CI"
+    notification_email: daniel.nguyen@intel.com
```

Comment:
Are you the long-term owner of these notifications? Is there an admin account that could receive these?